rotkehlchen/assets/converters.py

```diff
@@ -827,11 +832,17 @@ def asset_from_coinbase(cb_name: str, time: Optional[Timestamp] = None) -> Asset
         if time < COINBASE_DAI_UPGRADE_END_TS:
             # Then it should be the single collateral version
             return A_SAI
-        # else
         return A_DAI
 
-    # else
-    return symbol_to_asset_or_token(cb_name)
+    if not isinstance(cb_name, str):
+        raise DeserializationError(f'Got non-string type {type(cb_name)} for coinbase asset')
+
+    # Currently no unsupported assets
+    # if cb_name in UNSUPPORTED_CB_ASSETS:
+    #    raise UnsupportedAsset(cb_name)
```

Comment:
Is better to either remove it or add `UNSUPPORTED_CB_ASSETS` as an empty list. I would remove it and remove the error type from the comment